component/web/controller/src/main/java/org/exoplatform/web/WebAppController.java

```diff
@@ -332,8 +335,13 @@ public void service(HttpServletRequest req, HttpServletResponse res) throws Exce
                                 if (debug) {
                                     log.debug("Starting RequestLifeCycle for handler " + handler);
                                 }
-                                RequestLifeCycle.begin(ExoContainerContext.getCurrentContainer());
                                 started = true;
+                                ExoContainer container = ExoContainerContext.getCurrentContainer();
+                                if (!(container instanceof PortalContainer)) {
+                                  //We need this log to trace the issue for TA-423 if the bug still happen
+                                  log.error("current container is " + container + ", it should be a PortalContainer instance to start a portal request lifecycle");
```

Comment:
If it is an error, it should fail and the process should not continue.

What the customers/support can do with such error/warn logs ? The purpose is only to help us debug a problem.